canonical-websites/cn.ubuntu.com
# Upgrade markup for /about to Vanilla v1

## Done

Upgrade markup for /about to Vanilla v1

## QA

- Pull code
- Run `./run serve`
- View http://0.0.0.0:8010/about/
- Check layout works

## Issue / Card

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/64
